### PR TITLE
Pinecone Place App

### PR DESCRIPTION
**Pull request by FuadNabi**
Create front end for app

### CODE REVIEW

File: src/components/AddRoomImg.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function AddRoomImg() {
```
Comment (Meltrust) [inline src/components/AddRoomImg.js:3]: - [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const AddRoomImg= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.


image

That way, your code will meet the latest standards. 💪

File: src/components/DeleteRoom.js [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  }, [dispatch]);
+  const roomData = useSelector((state) => state.room.data);
+  const [error, setError] = useState(null);
+  const headers = {
+    'Content-Type': 'application/json',
+    Authorization: `Bearer ${localStorage.getItem('token')}`,
+  };
+
+  const handleDeleteRoom = async (id) => {
+    await fetch(`http://127.0.0.1:3000/api/v1/rooms/${id}`, {
```
Comment (Meltrust) [inline src/components/DeleteRoom.js:21]: - [ ] In the backend readme, you are instructing your users to run Rails on port 3001:

image

However, your frontend app expects port 3000:

image

Kindly correct this error in the backend readme, so your users can reach the backend. 👍

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @FuadNabi,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!






### To highlight:

-  Tests are passing✔️
-  Nice code organization ✔️ 
-  API is working well✔️ 
-  Super nice readme ✔️ 
-  Beautiful frontend ✔️ 



#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._







Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

**Comment by jorgegoco** [comment]
Hi Miguel,

This is Jorge. The requested changes were not shown up in the code review,
it only appears the suggestion of deleting the Test folder, so I went
ahead, merged everything, and asked for the evaluation. What should we do
to resolve this issue?

Sincerely,
Jorge

**Comment by Meltrust** [comment]
Hey @jorgegoco,  No worries.  I would advise you to fix the port, so the final evaluator does not get confused/blocked by it (I was).  This is because on your backend readme your users are instructed to run Rails at port 3001, but your frontend app expects port 3000.  You could add the commit directly to `main` on the backend repo.  You just need to change the port in the backend readme instructions from "3001" to "3000", or just "rails server" which defaults to port 3000.

About the arrow functions in the frontend, it's not a big deal you can do it whenever you have the time.

Also, kindly remember that in this special kind of review, everything is just suggested.

I hope this helps, and makes sense to you.

**Comment by jorgegoco** [comment]
Thanks a lot Miguel, I made the commit directly in GitHub back-end Readme
file, in dev and main branches.

Un saludo,

Jorge

**Pull request merged**